en/guide/toolchain.md

```diff
@@ -15,31 +16,34 @@ Below we explain how to set up a minimal build setup (**CMakeLists.txt**) file f
 
 The sections below show how you can set up the file for when the SDK is [installed system wide](../contributing/build.md#sdk_system_wide_install) (the default) or [locally](../contributing/build.md#sdk_local_install).
 
-> **Warning** SDK system-wide installation is not supported on Windows (see [#155](https://github.com/Dronecode/DronecodeSDK/issues/155)). 
+> **Warning** SDK system-wide installation is not supported on Windows (see [#155](https://github.com/Dronecode/DronecodeSDK/issues/155)).
 >  Instead build the app using a [local SDK installation](#sdk_local_install).
 >
 >  Windows gurus, we'd [love your help](../README.md#getting-help) to implement this).
 
 
 ### SDK Installed System-wide {#sdk_installed_system_wide}
 
-A "template" **CMakeLists.txt** is shown below. 
+A "template" **CMakeLists.txt** is shown below.
 Most of file is boilerplate - the main things you need to change are *your_project_name*, *your_executable_name* and *your_source_file*. You should also make sure that any plugins used by your app are listed in the `target_link_libraries` section.
 
 ```cmake
 cmake_minimum_required(VERSION 2.8.12)
 
-## Specify your project's name
+# Specify your project's name
 project(your_project_name)
 
+# Specify at least C++11
```

Comment:
Can we get this new stuff added to the examples?

Reply:
We should.

Reply:
Done in https://github.com/Dronecode/DronecodeSDK/pull/492

This is IMO otherwise good to go.